alumet-X.X/usr/bin/alumet-agent

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
now that we have the rpm package, this could become a link to the existing alumet-agent script (as discussed with Titouan)